### DISCUSSION TITLE
RustDesk on RaspberryPi

### DISCUSSION THREAD

**Discussion by Stefan1121** [rustdesk/rustdesk#3999] [General]
Hello, I installed RustDesk on my RaspberryPi4. 
Unfortunately, when I try to start the application, nothing happens, the program does not start.
Linux raspberrypi 5.10.103-v8+ #1529 SMP PREEMPT Tue Mar 8 12:26:46 GMT 2022 aarch64 GNU/Linux

Can someone help me?
Best regards from Germany.
Stefan

**Comment by Stefan1121** [comment]
If i type sudo rustdesk there is an error:
rustdesk: /lib/arm-linux-gnueabihf/libm.so.6: version `GLIBC_2.29' not found (required by rustdesk)
rustdesk: /lib/arm-linux-gnueabihf/libc.so.6: version `GLIBC_2.29' not found (required by rustdesk)

**Comment by WhiteAls** [comment]
I'm not an expert on Raspberry Pi, but based on what's written, it seems that you're trying to run Rustdesk version 1.1.9, which is compiled for the armhf (armv7) architecture, on Raspbian 5.10.103, which is compiled for the arm64 (aarch64) architecture.

I would recommend using the Rustdesk nightly builds, which are already compiled for the aarch64 architecture.

Alternatively, if you want to use Rustdesk version 1.1.9 specifically, try the following command

```
sudo apt-get update
sudo apt-get install libc6:armhf libstdc++6:armhf
```
